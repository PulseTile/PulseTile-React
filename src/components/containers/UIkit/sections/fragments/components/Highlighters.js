import React from 'react';

/**
 * This component returns content of Highlighters section
 */
const Highlighters = () => {
  return (
    <div id="highlighter" className="ui-section">
      <strong className="ui-title">Highlighters</strong>
      <div className="form-group-wrapper">
        <div className="row">
          <div className="col-xs-4">
            <div className="highlighter-wrapper">
              <span className="highlighter-success"></span>
              <div className="ui-highlighter"></div>
            </div>
          </div>
          <div className="col-xs-4">
            <div className="highlighter-wrapper">
              <span className="highlighter-warning"></span>
              <div className="ui-highlighter"></div>
            </div>
          </div>
          <div className="col-xs-4">
            <div className="highlighter-wrapper">
              <span className="highlighter-danger"></span>
              <div className="ui-highlighter"></div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default Highlighters;
