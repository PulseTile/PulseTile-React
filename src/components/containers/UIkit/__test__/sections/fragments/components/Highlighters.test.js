import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Highlighters from '../../../../sections/fragments/components/Highlighters';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Highlighters />', () => {
  it('should renders Highlighters with props correctly', () => {
    const component = shallow(<Highlighters />);
    expect(component).toMatchSnapshot();
  });
});
