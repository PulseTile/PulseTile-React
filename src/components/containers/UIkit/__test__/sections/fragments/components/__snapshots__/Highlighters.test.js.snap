// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Highlighters /> should renders Highlighters with props correctly 1`] = `
<div
  className="ui-section"
  id="highlighter"
>
  <strong
    className="ui-title"
  >
    Highlighters
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="row"
    >
      <div
        className="col-xs-4"
      >
        <div
          className="highlighter-wrapper"
        >
          <span
            className="highlighter-success"
          />
          <div
            className="ui-highlighter"
          />
        </div>
      </div>
      <div
        className="col-xs-4"
      >
        <div
          className="highlighter-wrapper"
        >
          <span
            className="highlighter-warning"
          />
          <div
            className="ui-highlighter"
          />
        </div>
      </div>
      <div
        className="col-xs-4"
      >
        <div
          className="highlighter-wrapper"
        >
          <span
            className="highlighter-danger"
          />
          <div
            className="ui-highlighter"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
